app/models/form.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: forms
+#
+#  id              :bigint           not null, primary key
+#  description     :text
+#  name            :string
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint
+#
+# Indexes
+#
+#  index_forms_on_organization_id  (organization_id)
+#
+class Form < ApplicationRecord
+  has_many :form_questions, dependent: :destroy
+  has_many :questions, through: :form_questions
+  has_many :submissions, dependent: :destroy
+
+  belongs_to :organization
```

Comment:
Quick question - per Mae's sketch `Form` belongs to `FormQuestion`, which would belong to `Organization`. Am I missing something?